.jenkins

```diff
@@ -2,10 +2,11 @@
 
 pipeline {
 
-  agent any
+  agent {
```

Comment:
Not for this PR, but would it be beneficial if we had a "goreleaser" pipeline that templated this out? does the goreleaser command invocation vary much between the pipelines that consume it?

Reply:
Its something we could look into. The goreleaser invocation doesn't vary much, but some of the other stages do depending on whats required for testing, so I don't think all stages could necessarily be pipelined out